websites/M/Manga Livre/dist/metadata.json

```
@@ -4,14 +4,20 @@
     "name": "Bas950",
     "id": "241278257335500811"
   },
+  "contributors": [
+    {
+      "name": "Aya",
+      "id": "256422547556401152"
+    }
+  ],
   "service": "Manga Livre",
   "description": {
     "en": "Read online and organize your readings here. Read in Portuguese (PT-BR) the most read manga like Naruto, Fairy Tail, One Piece and more!",
     "nl": "Lees online en organiseer uw lezings hier. Lees in het Portugees (PT-BR) de meest gelezen manga zoals Naruto, Fairy Tail, One Piece en meer!",
     "pt_BR": "Leia online e organize suas leituras aqui. Leia em português (PT-BR) os mangás mais lidos como Naruto, Fairy Tail, One Piece e outros!"
   },
   "url": "mangalivre.net",
-  "version": "1.3.9",
+  "version": "1.4.0",
```

This is just a fix, bump patch instead 
```suggestion
  "version": "1.3.10",
```